.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,46 @@
+
+name: Docker Image CI
+
+on:
+  push:
+    tags:
+      - v*
+    branches:
```

Comment:
I think disabling `master` branch build might be better. It doesn't make sense to put `master` branch build as `latest`.